attestation-verifier/src/pkg/lib/common/middleware/durationlog.go

```diff
@@ -61,8 +62,10 @@ func (w *logResponseWriter) WriteHeader(code int) {
 
 func (w *logResponseWriter) Write(body []byte) (int, error) {
 	// Sanitize the user-provided value to prevent XSS
-	sanitizedBody := bytes.ReplaceAll(body, []byte("<"), []byte("&lt;"))
-	sanitizedBody = bytes.ReplaceAll(sanitizedBody, []byte(">"), []byte("&gt;"))
+	bodyStr := string(body)
+	sanitizedBodyStr := html.EscapeString(bodyStr)
+	sanitizedBody := []byte(sanitizedBodyStr)
```

Comment:
How does the string formats updated? Did we check the logs?

Reply:
I didn't see any specific regression in my testing on the VM environment that was provided.  EscapeString() escapes only five characters: <, >, &, ' and ".